machines/wpia-packit-dev.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs, config, lib, self, ... }:
+{
+  imports = [
+    ./common/hardware-configuration.nix
+    ./common/disk-config.nix
+    ./common/base.nix
+    ./common/services.nix
+  ];
```

Comment:
perhaps it pushes the abstraction too far but i wonder if we have a default.nix in `./common` which imports all of these and then just do `imports = [ ./common ]`, i think something like that would work but not sure

Reply:
Having base and services split really helps when provisioning a new machine. I can import just base, deploy, check all is good, then deploy again with services.

Of course I could do that even if a `default.nix` existed, by simply not importing it, but this seems clearer.

I did try importing the hardware and disk config from base but ran into issues with the integration tests (which need base, but are a different kind of "hard"ware).